packages/overlay/src/overlay-trigger.ts

```diff
@@ -180,48 +116,39 @@ export class OverlayTrigger extends LitElement {
     }
 
     private onClickSlotChange(event: Event): void {
-        /* istanbul ignore if */
-        if (!event.target) {
-            return;
-        }
-        const slot = event.target as HTMLSlotElement;
-        const content = this.extractSlotContent(slot);
-
-        if (content) {
-            this.clickContent = content;
-        }
+        const content = this.extractSlotContentFromEvent(event);
+        this.clickContent = content || undefined;
     }
 
     private onHoverSlotChange(event: Event): void {
+        const content = this.extractSlotContentFromEvent(event);
+        this.hoverContent = content || undefined;
+    }
+
+    private onTargetSlotChange(event: Event): void {
+        const content = this.extractSlotContentFromEvent(event);
+        this.targetContent = content || undefined;
+    }
+
+    private extractSlotContentFromEvent(event: Event): HTMLElement | null {
         /* istanbul ignore if */
         if (!event.target) {
-            return;
+            return null;
```

Comment:
Doesn't `return;` inherently return `undefined` preventing the need to clearly define the fallback in the three methods above? If not, should we explicitly return `undefined` for the same reason?

Reply:
Yep, `return` is the same as `return undefined.` I new that for JS, but just worked that out for TS in this PR. I tidied this up.